README.md

```diff
@@ -246,7 +246,51 @@ makes it very easy to compare edits.
 1. After issuing a PR, other developers can view your edits, add comment and eventually decide whether to pull in your PR.
 1. You can always update your PR by pushing new edits to the same branch under your fork.
 
-# Step 7: Merge after PR is pulled in
+# Step 7: Rebasing 
+
+### Command Line
+
+1. Make sure you've fetched upstream master
+   
+   ```
+   git fetch upstream master
+   ```
+1. Rebase in interactive mode
+
+   ```
+   git rebase upstream/master -i
+   ```
+1. If you have messy commits, flag them with the squash thing. If there's only one
+conflicing commit, just `ctrl-x` out of the interactive window/message. You 
+should see an error message saying that the patch couldn't be applied.
+
+1. Check where the conflict happened
+
+   ```
+   git status
+   ```
+1. Open the conflicting file with the diff >>>>>>>/======= things.
+
```

Comment:
maybe like?

Open the conflicting file with your favorite editor and search for `>>>>>>>` or `=======`